src/style.css

```diff
@@ -39,7 +39,7 @@
 }
  
 html {
-  font-family: "Open Sans", "Helvetica Neue", Helvetica, Arial, sans-serif
```

Comment:
Do we also need to download `Inter`? (Add it to a link tag etc? 👀)

Reply:
✅ Done, please see my comment below